DEVELOPER.rst

```diff
@@ -1,71 +1,96 @@
-=================
- Install package
-=================
+====================================
+Setting up a development environment
+====================================
 
-Install in develop mode to notice changes immediately.
+Download code
+-------------
 
-.. code::
+First clone the repository to your local machine::
 
-   python setup.py develop
+    $ git clone https://github.com/bio-phys/MDBenchmark.git
+
+Create a virtual environment
```

Comment:
this is a conda environment. virtual environments are different for python. I would also link to the conda docs.